src/routes/+page.server.ts

```diff
@@ -1,16 +1,49 @@
 import type { PageServerLoad } from './$types';
-import type { ArticleEntry } from '$lib/types';
+
+import { D1CacheName, cacheGet, cachePut } from '$lib/storage';
+import type { ArticleEntry, CacheResponse } from '$lib/types';
 import { lastMonth } from '$lib/time';
 
+import { error } from '@sveltejs/kit';
+
 export const load: PageServerLoad = async ({ platform, url }) => {
-	const after = url.searchParams.get('after') ??  lastMonth.toISOString().split('T')[0];
+	if (platform === undefined) {
+		error(404, { message: 'Platform is undefined' });
```

Comment:
This might make more sense as a 400 error. I think you have it as a 400 error in the other page.

Reply:
omg so true